Dockerfile

```diff
@@ -7,7 +7,8 @@ WORKDIR $PROJECT_ROOT
 
 COPY requirements.txt requirements.txt
 RUN pip install -r requirements.txt
-COPY . /app
+COPY . .
 
-RUN sed -i -e 's/{{ label }}/Submit/g' /usr/local/lib/python3.6/site-packages/govuk_forms/templates/govuk_forms/submit-button.html
-CMD python3 django_reference_app/manage.py runserver 0.0.0.0:8000
+EXPOSE 8000
+
+CMD python3 ./manage.py runserver 0.0.0.0:8000
```

Comment:
The act of moving the `manage.py` up a level will break the job-migration manifest. This can be easily fixed after the merge however.